docs/operations-api-tenant-level-only.md

```diff
@@ -0,0 +1,21 @@
+# OperationsApiTenantLevelOnly
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Operations-V1-02
+
+## Description
+
+GET operations API MUST be scoped tenant-wide. Operations should _not_ vary per subscription.
```

Comment:
>GET [](http://example.com/codeflow?start=0&length=3)

does this refer to "get" verb?

Reply:
Yes, is there something you think I should change @AkhilaIlla?